src/web/components/Dateline.tsx

```diff
@@ -17,18 +17,89 @@ const dateline = css`
     margin-bottom: 6px;
 `;
 
-const description = css`
-    ${visuallyHidden}
+const secondaryDateWidth = css`
+    ${from.leftCol} {
+        width: 90%;
+    }
 `;
 
+// Checkbox functionality
+const toggleClass = css`
+    input[type='checkbox'] {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        opacity: 0;
+        display: none;
+    }
+
+    input[type='checkbox']:checked ~ span ~ p {
+        max-height: 80px;
+    }
+`;
+
+const pCheckboxStyle = css`
+    p {
+        max-height: 0;
+        overflow: hidden;
+        transition: max-height 0.4s ease;
+    }
+`;
+
+const labelStyles = css`
+    label {
+        cursor: pointer;
+    }
+    label {
+        position: relative;
+        display: block;
+    }
+    label:hover {
+        text-decoration: underline;
+    }
+    label::before {
+        content: '';
+        position: absolute;
+        width: 0;
+        height: 0;
+        top: 50%;
+        left: 10px;
+    }
+`;
+
+// At the moment there is no way to tell from primaryDateline and secondaryDatline
+// what the prepend should be (either 'Last modified on' or 'First published on ')
 export const Dateline: React.FC<{
-    dateDisplay: string;
+    primaryDateline: string;
     descriptionText: string;
-}> = ({ dateDisplay, descriptionText }) => (
-    <>
-        <div className={dateline} role="textbox">
-            <span className={description}>{descriptionText} </span>
-            {dateDisplay}
+    secondaryDateline?: string;
+}> = ({ primaryDateline, secondaryDateline }) => {
+    return (
+        <div className={dateline}>
+            {primaryDateline !== secondaryDateline ? (
+                <div
+                    className={cx(
+                        toggleClass,
+                        pCheckboxStyle,
+                        labelStyles,
+                        dateline,
+                    )}
+                >
+                    <input type="checkbox" id="dateToggle" />
+                    <span>
+                        <label htmlFor="dateToggle">{primaryDateline}</label>
+                    </span>
+                    {secondaryDateline && (
```

Comment:
I think we would want this conditional check higher up, as we don't want the checkbox etc when we don't have a secondaryDateLine to show

Reply:
I think as it's currently *not* optional, you can remove the optionality in the props though and rely on the `!==` check for primary and secondary date lines.

Reply:
In storybook, I am seeing a hover state when there's no secondaryDateLine in the markup though https://5dfcbf3012392c0020e7140b-pfylxkkawk.chromatic.com/?path=/story/components-articlemeta--article-story because it's not in the fixtures but primary !== secondary then becomes true